server/src/main/kotlin/org/javacs/kt/CompilerClassPath.kt

```diff
@@ -14,7 +14,7 @@ import java.nio.file.Path
  * Manages the class path (compiled JARs, etc), the Java source path
  * and the compiler. Note that Kotlin sources are stored in SourcePath.
  */
-class CompilerClassPath(private val config: CompilerConfiguration) : Closeable {
+class CompilerClassPath(private val config: CompilerConfiguration, private val gradleHome: StringBuilder) : Closeable {
```

Comment:
Hm, I feel like we shouldn't deal with Gradle-specifics at the 'higher' class path level. Perhaps we could rename `GradleConfiguration` to `ClassPathConfiguration` and `home` to `gradleHome`? Or perhaps with an additional layer by having a `GradleConfiguration` under `ClassPathConfiguration` (i.e. by moving the setting under `kotlin.classPath.gradle.home`)?